content/_partials/toptoolbar.html.haml

```diff
@@ -103,6 +103,8 @@
             - if page.url && page.url.match(/\/projects\/([^\/]+)\/$/) && page.layout == "project"
               %a.dropdown-item.feature{:href => expand_link(page.url)}
                 = page.title
+          %a.dropdown-item.feature{:href => expand_link('projects/document-jenkins-on-kubernetes')}
```

Comment:
I don't think we should pass through a redirect to reach the project page.  Let's link directly to the page.
```suggestion
          %a.dropdown-item.feature{:href => '/sigs/docs/gsod/2020/projects/document-jenkins-on-kubernetes/'}
```

Reply:
All other links in that file use expand_link, so maybe this one should be
`expand_link('sigs/docs/gsod/2020/projects/document-jenkins-on-kubernetes/')`
(the advantage of `expand_link` over absolue URL is that it supports running the site in a subfolder, e.g. for github page test deploy)

Reply:
That's good advice @zbynek .  Using `expand_link` worked when I tested it.